src/MountedTraversal.js

```diff
@@ -6,13 +6,16 @@ import {
   isCompoundSelector,
   AND,
 } from './Utils';
-import {
+import reactCompat from './react-compat';
+const {
   isDOMComponent,
   isCompositeComponent,
   isCompositeComponentWithType,
   isElement,
   findDOMNode,
-} from './react-compat';
+} = reactCompat;
```

Comment:
babel 5 incorrectly allows destructuring on a default import - the spec does not.